File: src/test/java/mobi/hsz/idea/gitignore/inspections/DuplicateEntryInspectionTest.java
package mobi.hsz.idea.gitignore.inspections;

import mobi.hsz.idea.gitignore.IgnoreBundle;
import mobi.hsz.idea.gitignore.codeInspection.IgnoreDuplicateEntryInspection;

public class DuplicateEntryInspectionTest extends InspectionTestCase {
    @Override
    public void setUp() throws Exception {
        super.setUp();
        myFixture.enableInspections(IgnoreDuplicateEntryInspection.class);
    }

    public void testSimpleCase() throws Exception {
        doHighlightingFileTest();
    }
    
    public void testSimpleCaseWithQuickFix() throws Exception {
        doHighlightingFileTestWithQuickFix(IgnoreBundle.message("quick.fix.remove.entry"));
    }
}

File: src/test/kotlin/mobi/hsz/idea/gitignore/inspections/DuplicateEntryInspectionTest.kt
// Copyright 2000-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.

package mobi.hsz.idea.gitignore.inspections

import mobi.hsz.idea.gitignore.IgnoreBundle
import mobi.hsz.idea.gitignore.codeInspection.IgnoreDuplicateEntryInspection

class DuplicateEntryInspectionTest : InspectionTestCase() {

    @Throws(Exception::class)
    public override fun setUp() {
        super.setUp()
        myFixture.enableInspections(IgnoreDuplicateEntryInspection::class.java)
    }

    @Throws(Exception::class)
    fun testSimpleCase() {
        doHighlightingFileTest()
    }

    @Throws(Exception::class)
    fun testSimpleCaseWithQuickFix() {
        doHighlightingFileTestWithQuickFix(IgnoreBundle.message("quick.fix.remove.entry"))
    }
}
